content/courses/connecting-to-offchain-data/oracles.md

```diff
@@ -303,85 +300,84 @@ Some relevant fields and configurations on the `AggregatorAccountData` type are:
-Each job listed defines the source the oracles will fetch data from and the
-weighting of each source. You can view the actual API endpoints that provide the
-data for this specific feed. When determining what data feed to use in your
-program, things like this are very important to consider.
+Each job defines the data sources the oracles fetch from and the weight assigned
+to each source. You can view the actual API endpoints that provide the data for
+this feed. When selecting a feed for your program, these considerations are key.
 
-Below is a two of the jobs related to the BTC_USD feed. It shows two sources of
-data: [MEXC](https://www.mexc.com/) and [Coinbase](https://www.coinbase.com/).
+Below are two of the jobs related to the BTC_USD feed, showing data from
+[MEXC](https://www.mexc.com/) and [Coinbase](https://www.coinbase.com/).
 
 ![Oracle Jobs](/public/assets/courses/unboxed/oracle-jobs.png)
 
-Once you’ve chosen a feed to use, you can start reading the data in that feed.
-You do this by simply deserializing and reading the state stored in the account.
-The easiest way to do that is by making use of the `AggregatorAccountData`
-struct we defined above from the `switchboard_v2` crate in your program.
+Once you’ve chosen a feed, you can start reading the data from that feed by
+deserializing and reading the state stored in the account. The easiest way to do
+this is by using the `AggregatorAccountData` struct from the
+`switchboard_solana` crate in your program.
 
 ```rust
-// import anchor and switchboard crates
+// Import anchor and switchboard crates
 use {
     anchor_lang::prelude::*,
-    switchboard_v2::AggregatorAccountData,
+    switchboard_solana::AggregatorAccountData,
 };
 
 ...
 
 #[derive(Accounts)]
 pub struct ConsumeDataAccounts<'info> {
-	// pass in data feed account and deserialize to AggregatorAccountData
-	pub feed_aggregator: AccountLoader<'info, AggregatorAccountData>,
-	...
+  // Pass in data feed account and deserialize to AggregatorAccountData
+  pub feed_aggregator: AccountLoader<'info, AggregatorAccountData>,
+  ...
 }
 ```
 
-Notice that we use the `AccountLoader` type here instead of the normal `Account`
-type to deserialize the aggregator account. Due to the size of
-`AggregatorAccountData`, the account uses what's called zero copy. This in
-combination with `AccountLoader` prevents the account from being loaded into
-memory and gives our program direct access to the data instead. When using
-`AccountLoader` we can access the data stored in the account in one of three
-ways:
+Notice that we use the `AccountLoader` type here instead of the regular
+`Account` type to deserialize the aggregator account. Due to the size of
+`AggregatorAccountData`, the account uses zero-copy deserialization. This,
+combined with `AccountLoader`, prevents the account from being fully loaded into
+memory, allowing our program to access the data directly.
```

Comment:
This still feels vague. "We can't load the data, so we can load it directly"

I *think* this is what it means:

>  Zero copy,combined with `AccountLoader`, prevents the account from being fully loaded into
memory. Instead our program can access individual data items within the account directly.

If that's correct, please update the text to read something like this (in your own words, whatever makes this clearest).

If that's not correct, please reply here!

Reply:
Yes updated content to make it more clear.